### PR TITLE
Story 3

### PR DESCRIPTION
**Pull request by randy-darsh**
Related Issue: #25 

Summary: Added browse Items by category functionality.

Author(s): randy-darsh

### CODE REVIEW

File: config/initializers/friendly_id.rb [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# FriendlyId Global Configuration
```
Comment (limsammy) [inline config/initializers/friendly_id.rb:1]: Note: Not sure if we need this initializer, but leaving it just in case

**Comment by JoelLindow** [comment]
Really good work Randy. I'm proud of how far you've come.

File: spec/features/visitor/visitor_can_see_all_category_items_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe "Visitor can view category path" do
+	context "visitor visits /category_name" do
+    scenario "and sees all items" do
+      category = create(:category, title: "Guitar")
+      create_list(:item, 6, category: category)
+      # As a visitor
```
Comment (limsammy) [inline spec/features/visitor/visitor_can_see_all_category_items_spec.rb:8]: Reminder to remove comments in a refactor branch

File: app/controllers/categories_controller.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class CategoriesController < ApplicationController
+
+  def show
+    @category = Category.find_by(title: params[:id])
```
Comment (amhursh) [inline app/controllers/categories_controller.rb:4]: Good way to get the category with the Friendly id route

File: spec/features/visitor/visitor_can_see_all_category_items_spec.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+      # When I visit “/:CATEGORY_NAME” 
+      visit category_path(category)
+      # I see all items assigned to that category
+      expect(page).to have_css('.item_card', count: 6)
```
Comment (limsammy) [inline spec/features/visitor/visitor_can_see_all_category_items_spec.rb:12]: Good work keeping the css class names uniform, will help with styling.

File: config/routes.rb [hunk 5]
```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+	resources :items, only: [:index]
+	resources :categories, path: "/", only: [:show]
```
Comment (amhursh) [inline config/routes.rb]: This looks good to me. I like the simplicity

File: spec/features/visitor/visitor_can_see_all_category_items_spec.rb [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+describe "Visitor can view category path" do
+	context "visitor visits /category_name" do
+    scenario "and sees all items" do
+      category = create(:category, title: "Guitar")
+      create_list(:item, 6, category: category)
+      # As a visitor
+      # When I visit “/:CATEGORY_NAME” 
+      visit category_path(category)
+      # I see all items assigned to that category
```
Comment (amhursh) [inline spec/features/visitor/visitor_can_see_all_category_items_spec.rb:11]: Good test. We will have to remember to remove comments

**Pull request merged**